src/lib/helpers/templateSource.ts

```diff
@@ -0,0 +1,32 @@
+export type Template = {
+  key?: string; // template slug, e.g. "magic-portfolio"
+  frameworks?: Array<{ 
+    key?: string;
+    providerRootDirectory?: string;
+  }>; // e.g. [{ key: "nextjs", providerRootDirectory: "./nextjs/magic-portfolio" }]
+  providerOwner?: string; // e.g. "appwrite"
+  providerRepositoryId?: string; // e.g. "templates-for-sites" or "template-for-store"
+  repository?: string; // if API already returns full URL, use it
+  repoUrl?: string;    // alias some builds may use
+  source?: string;     // alias some builds may use
+};
```

Comment:
I think our Console SDK already got type for response of listTemplates, @ItzNotABug do you know what it is?

Reply:
yep, there's 
`Models.TemplateFunction` & `Models.TemplateSite` based on the resource.